firmware/motorcontroller2/code/app/RTOS/RTOS_componentSpecific.h

```diff
@@ -8,6 +8,11 @@
 #ifndef RTOS_COMPONENTSPECIFIC_H_
 #define RTOS_COMPONENTSPECIFIC_H_
 
+#define RTOS_TASK_STACK_SIZE_1000MS	(100U)
```

Comment:
What does this define mean? can only 100 messages be sent in the 1second rtos task?

Reply:
This has nothing to do with CAN message. It's a parameter for the RTOS task creation. You have to specify how big you want the stack to be.  https://www.freertos.org/a00125.html. It's the parameter `usStackDepth`